pkg/actors/actors.go

```diff
@@ -1210,21 +1210,19 @@ func (a *actorsRuntime) CreateReminder(ctx context.Context, req *CreateReminderR
 			return err
 		}
 
-		// TODO: change the 3rd party library to take our format
-		jobSchedule := "@every " + req.Period
 		internalScheduleJobReq := &schedulerv1pb.ScheduleJobRequest{
 			Job: &runtimev1pb.Job{
-				Name:     a.actorsConfig.AppID + "||" + jobName,
```

Comment:
Why is the app ID being removed from the name?

Reply:
actors are not bound to an appId - the same actor type can be served by multiple appIds. The reminder is bound to a given ActorType+ActorID.

Reply:
See above how the jobName is being composed.

Reply:
Updated to new jobName composition.